WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -574,7 +575,14 @@ public static String getLoginUrl(Blog blog) {
     public static String getDefaultUserAgent() {
         if (mDefaultUserAgent == null) {
             // TODO: use WebSettings.getDefaultUserAgent() after upgrade to API level 17+
-            mDefaultUserAgent = new WebView(getContext()).getSettings().getUserAgentString();
+            try {
+                // Catch AndroidRuntimeException that could be raised by the WebView() constructor.
+                // See https://github.com/wordpress-mobile/WordPress-Android/issues/3594
+                mDefaultUserAgent = new WebView(getContext()).getSettings().getUserAgentString();
```

Comment:
Apparently [it's not specific to 5.0+ devices](https://stackoverflow.com/questions/29575313/namenotfoundexception-webview/29809338#comment48094186_29809338), but we could use `WebSettings.getDefaultUserAgent()` in the catch block, but I don't know if it's going to raise the same kind of **weird** exception. Fill a new issue for 5.0?